.travis.yml

```diff
@@ -4,7 +4,7 @@ env:
   global:
     - CIBW_SKIP="cp34-* cp35-*"
     - CIBW_TEST_REQUIRES="tox"
-    - CIBW_TEST_COMMAND="tox --installpkg /tmp/built_wheel/*.whl -e codecov --workdir {project} -c {project}"
+    - CIBW_TEST_COMMAND="tox --installpkg wheelhouse/*.whl -e codecov --workdir {project} -c {project}"
```

Comment:
You don’t want to only run `-e codecov` tox environment, neither you want to run all the envlist.. but the specific python version that is currently active. I think `-e py` (or `-e py-cov`) is the shortcut for the generic `python` executable. I’m typing from my phone..